package.json

```diff
@@ -66,7 +66,7 @@
   "devDependencies": {
     "commander": "2.9.0",
     "eslint-plugin-fxa": "git+https://github.com/mozilla/eslint-plugin-fxa#master",
-    "fxa-auth-db-mysql": "git+https://github.com/mozilla/fxa-auth-db-mysql.git#master",
+    "fxa-auth-db-mysql": "git+https://github.com/mozilla/fxa-auth-db-mysql.git#remove-bufferization-magic-take-two",
```

Comment:
Note that this is a devDependency, so it's not currently in the shrinkwrap file, so it suffices to change the reference here.